src/inference/src/dev/iplugin.cpp

```diff
@@ -313,65 +331,109 @@ std::unordered_set<std::string> ov::get_supported_nodes(
+                        if (!supported.count(fused_model_op_map[name]) &&
+                            has_all_consumers_unsupported(supported, op)) {
+                            supported.erase(name);
+                            update_supported = true;
+                        }
                     }
                 }
             }
             // Calculate the data size that needs to be transmitted after the current model is split
-            int64_t total_len = 0;
-            for (auto& op : model->get_ordered_ops()) {
-                if (supported.count(op->get_friendly_name()) && !ov::op::util::is_constant(op) &&
-                    !ov::op::util::is_parameter(op)) {
-                    if (has_users_unsupported(supported, op)) {
-                        int64_t op_size = 1;
-                        for (size_t shape_id = 0; shape_id < op->get_output_partial_shape(0).size(); shape_id++) {
-                            if (!op->get_output_partial_shape(0)[shape_id].is_dynamic()) {
-                                int64_t len = op->get_output_partial_shape(0)[shape_id].get_length();
-                                if (len >= 1)
-                                    op_size *= len;
+            std::map<std::string, int> temp_pair_checker_2;
+            if (check_variables(temp_pair_checker_2, supported)) {
+                int64_t total_len = 0;
+                for (auto& op : model->get_ordered_ops()) {
+                    if (supported.count(op->get_friendly_name()) && !ov::op::util::is_constant(op) &&
+                        !ov::op::util::is_parameter(op)) {
+                        if (has_users_unsupported(supported, op)) {
+                            int64_t op_size = 1;
+                            for (size_t shape_id = 0; shape_id < op->get_output_partial_shape(0).size(); shape_id++) {
+                                if (!op->get_output_partial_shape(0)[shape_id].is_dynamic()) {
+                                    int64_t len = op->get_output_partial_shape(0)[shape_id].get_length();
+                                    if (len >= 1)
+                                        op_size *= len;
+                                }
                             }
+                            total_len += op_size;
                         }
-                        total_len += op_size;
                     }
                 }
-            }
-            if ((total_len < last_total_len || last_total_len == 0) && !cancel_split) {
-                last_total_len = total_len;
-                copy_set(supported, temp_supported_1);
-                copy_set(unsupported, temp_unsupported_1);
+                if ((total_len < last_total_len || last_total_len == 0) && !cancel_split) {
+                    last_total_len = total_len;
+                    copy_set(supported, temp_supported_1);
+                    copy_set(unsupported, temp_unsupported_1);
+                }
             }
```

Comment:
want to make sure, that do the changes in iplugin interface relevant to the issue of this PR?

Reply:
Hi @songbell , update the Details of this PR, please review.